### PR TITLE
Add gribi feature textproto

### PR DESCRIPTION
**Pull request by xw-g**

### CODE REVIEW

**Comment by bensons** [comment]
@xw-g

It appears that this PR is failing the Validate OpenConfig Paths check. The error seems to be:
  file: /home/runner/work/featureprofiles/featureprofiles/feature/gribi/feature.textproto
    line 36: missing from YANG "/network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/next-hop/state/id"
    line 63: missing from YANG "/network-instances/network-instance/afts/next-hops/next-hop/state/ip-in-ip/state/src-ip"
    line 66: missing from YANG "/network-instances/network-instance/afts/next-hops/next-hop/state/ip-in-ip/state/dst-ip"

File: feature/gribi/feature.textproto [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  path: "/network-instances/network-instance/afts/next-hops/next-hop/state/encapsulate-header"
+}
+telemetry_path {
+  path: "/network-instances/network-instance/afts/next-hops/next-hop/state/decapsulate-header"
+}
+telemetry_path {
+  path: "/network-instances/network-instance/afts/next-hops/next-hop/state/network-instance"
+}
+telemetry_path {
+  path: "/network-instances/network-instance/afts/next-hops/next-hop/state/ip-in-ip/state/src-ip"
```
Comment (dplore) [inline feature/gribi/feature.textproto:63]:
```suggestion
  path: "/network-instances/network-instance/afts/next-hops/next-hop/ip-in-ip/state/src-ip"
```

File: feature/gribi/feature.textproto [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+}
+telemetry_path {
+  path: "/network-instances/network-instance/afts/next-hops/next-hop/state/ip-in-ip/state/dst-ip"
```
Comment (dplore) [inline feature/gribi/feature.textproto:66]:
```suggestion
  path: "/network-instances/network-instance/afts/next-hops/next-hop/ip-in-ip/state/dst-ip"
```

File: feature/gribi/feature.textproto [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  path: "/network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/id"
+}
+telemetry_path {
+  path: "/network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/programmed-id"
+}
+telemetry_path {
+  path: "/network-instances/network-instance/afts/next-hop-groups/next-hop-group/state/backup-next-hop-group"
+}
+telemetry_path {
+  path: "/network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/next-hop/state/id"
```
Comment (dplore) [inline feature/gribi/feature.textproto:36]:
```suggestion
  path: "network-instances/network-instance/afts/next-hop-groups/next-hop-group/next-hops/next-hop/state/index"
```

**Review by dplore — CHANGES_REQUESTED** [review]

**Review by dplore — APPROVED** [review]

**Pull request merged**